packages/web-components/package.json

```diff
@@ -94,8 +94,8 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
     "@microsoft/fast-components-styles-msft": "^4.29.0",
-    "@microsoft/fast-element": "^0.16.0",
-    "@microsoft/fast-foundation": "^1.7.0",
```

Comment:
I think this *could* stay at 1.7.x - this will take 1.7.1 as acceptable.

Reply:
I was curious about that, I'll switch it back to ^1.17.0